wpimath/src/main/java/edu/wpi/first/math/Matrix.java

```diff
@@ -573,11 +573,10 @@ public <R2 extends Num, C2 extends Num> void extractFrom(
    * Decompose "this" matrix using Cholesky Decomposition. If the "this" matrix is zeros, it will
    * return the zero matrix.
    *
-   * @param lowerTriangular Whether or not we want to decompose to the lower triangular Cholesky
-   *     matrix.
+   * @param lowerTriangular Whether we want to decompose to the lower triangular Cholesky matrix.
    * @return The decomposed matrix.
-   * @throws RuntimeException if the matrix could not be decomposed(ie. is not positive
-   *     semidefinite).
+   * @throws RuntimeException if the matrix could not be decomposed(i.e. is not positive
+   *     semi-definite).
```

Comment:
The mathematical term does not have a dash (it's just "positive semidefinite").  This should also have a space before the opening paren.